db/schema.rb

```diff
@@ -136,6 +136,7 @@
     t.datetime "updated_at"
     t.boolean  "admin",                  limit: 1,   default: false
     t.boolean  "enabled",                limit: 1,   default: true
+    t.string   "ldap_name",              limit: 255, default: ""
```

Comment:
Do you think two different users can end up having the same `ldap_name`? I think we should add a unique index on it.